### PR TITLE
Merge patch-2 commits

### PR DESCRIPTION
**Pull request by Defrothew**

### CODE REVIEW

**Comment by Defrothew** [comment]
Sorry for creating scattered commits. This pull request is related to: https://github.com/growthbook/growthbook-php/pull/4

File: src/User.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ private function isIncluded(Experiment $experiment): bool
             }
         }
 
-        if ($experiment->url && !Util::urlIsValid($experiment->url)) {
+        if ($experiment->url && !Util::urlIsValid($experiment->url, $this->client->config->url)) {
```
Comment (jdorn) [inline src/User.php:63]: `$this->client->config->url` may be null, but `urlIsValid` is always expecting a string.  Can you either make sure this argument is always a string or change the function signature to accept nulls?

**Review by jdorn — COMMENTED** [review]
Hi @Defrothew !  Thanks for PRs.  Can you merge in the latest master commits to this one?  I just merged #3 and #4 which are required for this code to work.

**Comment by Auz** [comment]
Merged in this change in another branch.

**Pull request closed without merging**